source/css/nav_style.css

```diff
@@ -788,12 +790,12 @@ span.searcher-result-preview:last-child {
         display: none;
         position: absolute;
         right: 0;
-        width: 250px;
+        /*width: 250px;*/
```

Comment:
let's remove comments so that the people taking over from don't scratch their heads thinking "why have the developers not removed it completely; why just commented out?"